core/src/main/java/io/github/mabartos/level/RiskLevelCondition.java

```diff
@@ -46,6 +47,13 @@ public boolean matchCondition(AuthenticationFlowContext context) {
         final AuthenticatorConfigModel authConfig = context.getAuthenticatorConfig();
 
         if (authConfig != null) {
+            // Check if risk-based authentication is enabled
+            var riskEngine = context.getSession().getProvider(RiskEngine.class);
+            if (riskEngine != null && !riskEngine.isRiskBasedAuthnEnabled()) {
+                logger.debug("Risk-based authentication is disabled. Skipping risk level condition check.");
```

Comment:
```suggestion
                logger.warn("Risk-based authentication is disabled. Skipping risk level condition check.");
```

This is also an example where the condition is already in the used authentication flow. So, we should print a warning if the risk-based authn is disabled. 

We could also have some other warnings for it, but there might be a case when this condition is left in the authn flow on its own.